src/sentry/plugins/base/response.py

```diff
@@ -1,20 +1,24 @@
-from django.http import HttpResponse
+from __future__ import annotations
+
+from typing import Any
+
+from django.http import HttpRequest, HttpResponse
 from django.template.context_processors import csrf
 
-__all__ = ("Response",)
+from sentry.web.helpers import render_to_string
 
+__all__ = ("DeferredResponse",)
 
-class Response:
-    def __init__(self, template, context=None):
+
+class DeferredResponse:
```

Comment:
I renamed this because it was getting confused in several places with `rest_framework.response.Response` and `requests.models.Response` -- this name should also be more clear as to what it's trying to do